### DISCUSSION TITLE
Example online interface to Conformer-CTC model

### DISCUSSION THREAD

**Discussion by Slava715** [NVIDIA-NeMo/NeMo#2475] [Q&A]
Hello please add an example online interface to Conformer-CTC model.
I tried use this example from Quartznet: https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/02_Online_ASR_Microphone_Demo.ipynb
But this not worked from Conformer-CTC model.
Also on model Quartznet im view large loss quality on russian langue on this example.

**Comment by okuchaiev** [comment]
@VahidooX shouldn't citrinet based example work?

**Reply by Slava715** [reply]
I was able to run this example on a Conformer-CTC model by replacing:
     "for block in model_definition['JasperEncoder']['jasper']:
            timestep_duration *= block['stride'][0] ** block['repeat']"
fixed value:
     "timestep_duration = 0.02"

**Reply by Slava715** [reply]
It worked, but the quality is poor.
